lib/unifex/loader.ex

```diff
@@ -24,7 +24,7 @@ defmodule Unifex.Loader do
         arg_names = args |> Keyword.keys() |> Enum.map(&Macro.var(&1, nil))
 
         quote do
-          defnifp unquote(wrapped_name)(unquote_splicing(arg_names))
+          defnifp(unquote(wrapped_name)(unquote_splicing(arg_names)))
```

Comment:
This shouldn't happen as `defnifp` is listed in `locals_without_parens` in  Bundlex's formatter.exs

Reply:
fixed